src/app/InteractionModelEngine.h

```diff
@@ -95,22 +95,22 @@ class InteractionModelEngine : public Messaging::ExchangeDelegate, public Comman
     Messaging::ExchangeManager * GetExchangeManager(void) const { return mpExchangeMgr; };
 
     /**
-     *  Creates a new read client and send ReadRequest message to the node using the read client,
+     *  Creates a new read client and send ReadRequestMessage message to the node using the read client,
      *  shutdown if fail to send it out
      *
      *  @retval #CHIP_ERROR_NO_MEMORY If there is no ReadClient available
      *  @retval #CHIP_NO_ERROR On success.
      */
-    CHIP_ERROR SendReadRequest(ReadPrepareParams & aReadPrepareParams, ReadClient::Callback * aCallback);
+    CHIP_ERROR SendReadRequestMessage(ReadPrepareParams & aReadPrepareParams, ReadClient::Callback * aCallback);
```

Comment:
I'm not sure the renaming is the right thing here.  The idea here is to send a read request action, not just a message...

Reply:
sure, rename it back.